.circleci/config.yml

```diff
@@ -1,5 +1,8 @@
 version: 2.1
 
+orbs:
+  gh-pages: sugarshin/gh-pages@volatile
```

Comment:
Beaware that sugarshin has access to all secrets of this project :)

Reply:
Oh, I wasn't aware of that :scream: 
Then it's better to just copy their orb source into our config?

Reply:
Ok, I added the relevant parts to our jobs, now it is more clear what happens and we don't depend on the third-party orb anymore.